README.md

```diff
@@ -26,89 +76,277 @@ LibRealSense supported version: v2.51.1 (see [realsense2_camera release notes](h
       ```bashrc
       git clone https://github.com/IntelRealSense/realsense-ros.git -b ros2-development
       cd ~/ros2_ws
       ```
-### Step 4: Install dependencies
+  
+  - Install dependencies
    ```bash
    sudo apt-get install python3-rosdep -y
    sudo rosdep init # "sudo rosdep init --include-eol-distros" for Eloquent and earlier
    rosdep update # "sudo rosdep update --include-eol-distros" for Eloquent and earlier
    rosdep install -i --from-path src --rosdistro $ROS_DISTRO --skip-keys=librealsense2 -y
    ```
 
-### Step 5: Build
+  - Build
    ```bash
    colcon build
    ```
 
-### Step 6: Terminal environment
+  -  Source environment
    ```bash
-   ROS_DISTRO=<YOUR_SYSTEM_ROS_DISTRO>  # set your ROS_DISTRO: humble, galactic, foxy, eloquent, dashing
+   ROS_DISTRO=<YOUR_SYSTEM_ROS_DISTRO>  # set your ROS_DISTRO: humble, galactic, foxy
    source /opt/ros/$ROS_DISTRO/setup.bash
    cd ~/ros2_ws
    . install/local_setup.bash
    ```
+  
+  </details>
+
+<hr>
+
+<h2 id="usage">
+  Usage
+</h2>
+
+<h3 id="start-camera-node">
+  Start the camera node
+</h3>
+  
+  #### with ros2 run:
+    ros2 run realsense2_camera realsense2_camera_node
+    # or, with parameters, for example - temporal and spatial filters are enabled:
+    ros2 run realsense2_camera realsense2_camera_node --ros-args -p enable_color:=false -p spatial_filter.enable:=true -p temporal_filter.enable:=true
+  
+  #### with ros2 launch:
+    ros2 launch realsense2_camera rs_launch.py
+    ros2 launch realsense2_camera rs_launch.py depth_module.profile:=1280x720x30 pointcloud.enable:=true
+
+<hr>
+
+<h3 id="parameters">
+  Parameters
+<h3>
+  
+### Sensor Parameters:
+- Each sensor has a unique set of parameters.
+- Video sensors, such as depth_module or rgb_camera have, at least, the 'profile' parameter.</br>
+  - The profile parameter is a string of the following format: \<width>X\<height>X\<fps> (The deviding character can be X, x or ",". Spaces are ignored.)
+  - For example: ```depth_module.profile:=640x480x30```
+- Since infra1, infra2 and depth are all streams of the depth_module, their width, height and fps are defined by their common sensor.
+- If the specified combination of parameters is not available by the device, the default configuration will be used.
 
-&nbsp;
+</br>
 
-## Usage Instructions
+### Available Parameters:
+- For the entire list of parameters type `ros2 param list`.
+- For reading a parameter value use `ros2 param get <node> <parameter_name>` 
+  - For example: `ros2 param get /camera/camera depth_module.emitter_on_off`
+- For setting a new value for a parameter use `ros2 param set <node> <parameter_name> <value>`
+  - For example: `ros2 param set /camera/camera depth_module.emitter_on_off true`
 
-### Start the camera node
-To start the camera node in ROS:
+#### Parameters that can be modified during runtime:

```

Comment:
Maybe we can use the "collapse" method more?
this section takes many lines and nor sure everyone needs it.
We can ether collapse or point to another doc if it's very big.

Same for the new CS chapter.
What do you think?

Reply:
we can use the collapse as much as we want.
The main reason I don't want to split it to another docs, is the keep it simple for searching.
Most of users search with CTRL+F, so it is easier to keep everything in one document.
What do you think ?

Reply:
Sure NP.
Does CTRL + F works inside "details"?

Reply:
As I tested it now, NO :)